CodeForAfricaLabs/CitizenReporter.Android
# Add Feedback page

#### What does this PR do?
Add Feedback page to the settings menu page
#### Description of Task to be completed?
Add a form to the feedback page
#### How should this be manually tested?
After cloning the repo, run the app on android studio, click the settings menu then click on feedback. This should display a form which users can input information related to their feedback
